### DISCUSSION TITLE
2005/03/01/%E5%A4%A7%E5%AE%B6%E8%AF%95%E8%AF%95-gmail-drive

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#6] [Announcements]
# 2005/03/01/%E5%A4%A7%E5%AE%B6%E8%AF%95%E8%AF%95-gmail-drive

http://www.puremango.co.uk/gdrive/index.php

https://blog.qizhen.xyz/2005/03/01/%E5%A4%A7%E5%AE%B6%E8%AF%95%E8%AF%95-gmail-drive

**Comment by ruanqizhen** [comment]
之前博客下的留言：
```
Qizhen
2005年03月1日 21:31
一点也不好用 :)
```
